client/App.jsx

```diff
@@ -1,21 +1,48 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import axios from 'axios';
+import styled from 'styled-components';
+import TopDetails from './TopDetails.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.className = props.className;
```

Comment:
In general I think the style guide suggests destructuring props in the function where they get used (as you've done with state below).